<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleManager</class>
 <widget class="QWidget" name="StyleManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>433</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="3" rowspan="2">
    <widget class="QWidget" name="styleTypeContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="paragButton">
        <property name="text">
         <string>P</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="charButton">
        <property name="text">
         <string>C</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="5" rowspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="StyleManagerWelcome" name="welcomePage"/>
     <widget class="ParagraphGeneral" name="paragraphStylePage"/>
     <widget class="CharacterGeneral" name="characterStylePage"/>
     <widget class="NewStyleWidget" name="createPage"/>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QWidget" name="selectorPane" native="true">
     <layout class="QGridLayout" name="gridLayout1">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="bNew">
        <property name="text">
         <string>&amp;New</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bDelete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="StylesWidget" name="styles"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParagraphGeneral</class>
   <extends>QWidget</extends>
   <header>dialogs/ParagraphGeneral.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NewStyleWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/NewStyleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CharacterGeneral</class>
   <extends>QWidget</extends>
   <header>dialogs/CharacterGeneral.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StylesWidget</class>
   <extends>QListWidget</extends>
   <header>dialogs/StylesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>StyleManagerWelcome</class>
   <extends>QWidget</extends>
   <header>dialogs/StyleManagerWelcome.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
